source/common/http/http2/codec_impl.cc

```diff
@@ -118,6 +122,40 @@ void ConnectionImpl::StreamImpl::encodeTrailers(const HeaderMap& trailers) {
     parent_.sendPendingFrames();
   }
 }
+void ConnectionImpl::StreamImpl::readDisable(bool disable) {
+  ENVOY_CONN_LOG(debug, "Stream {} disabled: disable {}, unconsumed_bytes {}", parent_.connection_,
+                 stream_id_, disable, unconsumed_bytes_);
+  if (disable) {
+    ++buffers_overrun_;
```

Comment:
nit: I would rename variable to read_disable_count_ to match what is going on here. (or maybe rename readDisable() to something else).